src/main/java/bio/terra/common/retry/transaction/TransactionRetryConfig.java

```diff
@@ -0,0 +1,121 @@
+package bio.terra.common.retry.transaction;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Objects;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.classify.BinaryExceptionClassifier;
+
+/**
+ * Configuration settings for how database transactions are retried. There are 2 classes of retries,
+ * fast and slow. Fast retries usually measure in milliseconds with a constant retry period (with
+ * some random jitter) and many attempts. Slow retries usually measure in seconds with an
+ * exponential back off and few attempts.
+ */
+@ConfigurationProperties(prefix = "terra.common.retry.transaction")
+public class TransactionRetryConfig implements InitializingBean {
```

Comment:
There is a pattern of separating properties from configuration that is used in TCL.
I'm not sure that should apply here. @gjuggler pioneered that split, I think, so maybe he can comment or provide a pointer.

I think if you provided defaults for the exceptions, you would not need to derive from InitializingBeam.